Source/Plugins/ImportExport/ImportExportFilters/NodesTrianglesToStl.cpp

```diff
@@ -193,8 +186,7 @@ void NodesTrianglesToStl::execute()
   {
 
     QString ss = QObject::tr("Error creating parent path '%1'").arg(getOutputStlDirectory());
-    setErrorCondition(-1);
-    notifyErrorMessage(getHumanLabel(), ss, getErrorCondition());
+    notifyErrorMessage("", ss, -1001);
```

Comment:
Is this intended to be different?  Where did -1001 come from?

Reply:
No, that must be an error from the grep.  I'll take care of it.